tools/packaging/common/envoy_bootstrap_v2.json

```diff
@@ -262,7 +262,11 @@
             ],
             "validation_context": {
               "trusted_ca": {
+                {{ if .sign_cert_at_kubernetes_ca }}
```

Comment:
I'm curious: istiod doesn't have a sidecar, right? Are you changing this for normal workloads?

Reply:
Yes, this is for normal workloads.